lib/components/Input/wheel-picker/time-wheel-picker/__tests__/time-wheel-picker.test.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { TimeWheelPicker } from "..";
+import { DropdownProvider } from "@providers/dropdown/dropdownProvider";
+import { useDropdown } from "@hooks/useDropdown";
+import useBreakpoints from "@hooks/useBreakpoints";
+import { act } from "react-dom/test-utils";
+
+jest.mock("@hooks/useDropdown", () => ({
+    useDropdown: jest.fn(),
+}));
+jest.mock("@hooks/useBreakpoints");
+
+class ResizeObserver {
+    observe() {}
+    unobserve() {}
+    disconnect() {}
+}
+
+global.ResizeObserver = ResizeObserver;
+const mockOnValueChange = jest.fn();
+
+const renderComponent = () =>
+    render(
+        <DropdownProvider>
+            <TimeWheelPicker
+                is12Hour={true}
```

Comment:
is12Hour

Reply:
![Screenshot 2024-08-15 at 12 16 45 PM](https://github.com/user-attachments/assets/4f30e76c-e3c9-45ee-985f-331d04b19010)
Also it seems to be true by default